File: auto-docs/executables/julia/percent_error_bar.jl
# Learn about API authentication here: plot.ly/julia/getting-started
# Find your api_key here: plot.ly/settings/api

using Plotly

Plotly.signin("TestBot", "r1neazxo9w")
data = [
  [
    "x" => [0, 1, 2], 
    "y" => [6, 10, 2], 
    "error_y" => [
      "type" => "percent", 
      "value" => 50, 
      "visible" => true
    ], 
    "type" => "scatter"
  ]
]
response = Plotly.plot(data, ["filename" => "percent-error-bar", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/matlab/percent_error_bar.m
signin('TestBot', 'r1neazxo9w')

data = {...
  struct(...
    'x', [0, 1, 2], ...
    'y', [6, 10, 2], ...
    'error_y', struct(...
      'type', 'percent', ...
      'value', 50, ...
      'visible', true), ...
    'type', 'scatter')...
};

response = plotly(data, struct('filename', 'percent-error-bar', 'fileopt', 'overwrite', 'auto_open', 'false'));
plot_url = response.url